utils/build-script

```diff
@@ -56,7 +56,8 @@ _SUPPORTED_XCODE_BUILDS = [
     ("12.3", "12C33"),
     ("12.4", "12D4e"),
     ("12.5", "12E262"),
```

Comment:
[Xcode 12.5.1 (12E507)][] is also available, as mentioned on the [forums][].

```suggestion
    ("12.5", "12E262"),
    ("12.5.1", "12E507"),
```

[Xcode 12.5.1 (12E507)]: <https://developer.apple.com/news/releases/?id=06212021>
[forums]: <https://forums.swift.org/t/build-swift-error-using-unsupported-xcode-version/50049>